staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/zz_generated.conversion.go

```diff
@@ -291,6 +301,46 @@ func Convert_url_Values_To_v1_CreateOptions(in *url.Values, out *CreateOptions,
 	return autoConvert_url_Values_To_v1_CreateOptions(in, out, s)
 }
 
+func autoConvert_url_Values_To_v1_DeleteOptions(in *url.Values, out *DeleteOptions, s conversion.Scope) error {
+	// WARNING: Field TypeMeta does not have json tag, skipping.
+
+	if values, ok := map[string][]string(*in)["gracePeriodSeconds"]; ok && len(values) > 0 {
+		if err := runtime.Convert_Slice_string_To_Pointer_int64(&values, &out.GracePeriodSeconds, s); err != nil {
+			return err
+		}
+	} else {
+		out.GracePeriodSeconds = nil
+	}
+	if values, ok := map[string][]string(*in)["preconditions"]; ok && len(values) > 0 {
+		// FIXME: out.Preconditions is of not yet supported type and requires manual conversion
+	} else {
+		out.Preconditions = nil
+	}
+	if values, ok := map[string][]string(*in)["orphanDependents"]; ok && len(values) > 0 {
+		if err := runtime.Convert_Slice_string_To_Pointer_bool(&values, &out.OrphanDependents, s); err != nil {
+			return err
+		}
+	} else {
+		out.OrphanDependents = nil
+	}
+	if values, ok := map[string][]string(*in)["propagationPolicy"]; ok && len(values) > 0 {
+		// FIXME: out.PropagationPolicy is of not yet supported type and requires manual conversion
```

Comment:
This has to be addressed to (will be similar to DeletionPropagation).

Reply:
addressed